### PR TITLE
Fix documentation: how to start server for cpp tests

### PR DESCRIPTION
**Pull request by candlerb**
### Modifications

Out-of-date instructions in pulsar-client-cpp/README.md about how to start test server

### CODE REVIEW

File: pulsar-client-cpp/README.md [hunk 1]
```diff
@@ -202,8 +202,7 @@ ${PULSAR_PATH}/pulsar-client-cpp/tests/
 
 # Execution
 # Start standalone broker
-export PULSAR_STANDALONE_CONF=${PULSAR_PATH}/pulsar-client-cpp/tests/standalone.conf
-${PULSAR_PATH}/bin/pulsar standalone
+${PULSAR_PATH}/pulsar-test-service-start.sh
```
Comment (merlimat) [inline pulsar-client-cpp/README.md]: We could also mention `pulsar-test-service-stop.sh` to tear down the service

Comment (candlerb) [reply pulsar-client-cpp/README.md]: Added, and rebased to master and repushed.

**Comment by sijie** [comment]
run java8 tests
run integration tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Comment by Jennifer88huang** [comment]
run java8 tests

**Comment by wolfstudy** [comment]
run java8 tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Comment by sijie** [comment]
run cpp tests

**Comment by sijie** [comment]
run java8 tests

**Pull request merged**